From cf66bf1737bd21323f6a30528f972d2667d9ff5d Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Mon, 26 Nov 2018 10:22:43 +0900
Subject: [PATCH 514/909] arm64: dts: r8a77990-ebisu: Enable other VIN

commit 27c04d35a98453929ff413c50295d68042aaa1b8 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

The connection between CSI2 and VIN can be switched.
Therefore, this patch enables VIN5.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts b/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
index f48c40b3562c..3dd3c61c3621 100644
--- a/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
@@ -703,6 +703,10 @@
 	status = "okay";
 };
 
+&vin5 {
+	status = "okay";
+};
+
 &xhci0 {
 	pinctrl-0 = <&usb30_pins>;
 	pinctrl-names = "default";
-- 
2.17.1

